scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -97,6 +98,10 @@
 	private final List<CipherSuite> preferredCipherSuites;
 
 	protected Integer maxFragmentLengthCode;
+	
+	private PreSharedKeyIdentity pskIdentity;
+	
+	private byte[] psk;
```

Comment:
these both looks like "two return values" from `configurePskCredentials`.
Maybe you find a other solution.

Reply:
do you have any ideas or pattern to return two values from one method?

Reply:
You may add a "PskUtil" which has a constructor implementing the `configurePskCredentials`.
Then use that PskUtil instance only as local variable.

Reply:
PskUtil should then contain the two above fields psk and pskIdentity.